seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/DateTimeUtils.java

```diff
@@ -49,10 +51,24 @@ public static LocalDateTime parse(String dateTime, Formatter formatter) {
         return LocalDateTime.parse(dateTime, FORMATTER_MAP.get(formatter));
     }
 
+    public static LocalDateTime parse(long timestamp) {
```

Comment:
Would you mind add some test case for this utils? It be used in lots of place, but without test case.

Reply:
Done.